app/controllers/tasks_controller.rb

```diff
@@ -1,4 +1,5 @@
 class TasksController < ApplicationController
   def index
+    @tasks = Task.where(status:1)
+    @yettasks = Task.where(status:0)
```

Comment:
順番逆が良さそうです！(0が上にある方が自然)

変数名は
未着手 => tasks_not_yet
終了済 => tasks_done
などが分かりやすそうですね！

tasksだと未着手、着手済みが全て含まれているように感じます 🤔 
```suggestion
    @tasks_not_yet = Task.where(status:0)
    @tasks_done = Task.where(status:1)
```